products/bash/release-ar.sh

```diff
@@ -14,7 +14,7 @@
 #
 # PARAMETERS:
 #   -n : <namespace> (string), Defaults to "cp4i"
-#   -r : <release-name> (string), Defaults to "demo"
+#   -r : <release-name> (string), Defaults to "ar-demo"
```

Comment:
Suggest keeping this as `demo` and passing `-r ar-demo` to this and ar_remote_create.sh when calling from install.sh to make sure the same name is used for both.

```suggestion
#   -r : <release-name> (string), Defaults to "demo"
```